README.rst

```diff
@@ -35,19 +35,80 @@ In the Kinto project settings
     kinto.attachment.keep_old_files = true
 
 
+The ``gzipped`` option
+----------------------
+
 If you want uploaded files to get gzipped when stored:
 
 .. code-block:: ini
 
     kinto.attachment.gzipped = true
 
 
+The ``use_content_encoding`` option
+-----------------------------------
+
+If you want uploaded files to be Gzipped and automatically unzipped with S3 you can activate the Content-Encoding header:
```

Comment:
automatically unzipped when? just before being stored on S3? When downloaded? 

I would not mention the word header here, it's confusing for someone that doesn't know what it means. Just `content encoding setting` maybe?